frontend/src/Series/Details/SeriesDetailsLinks.tsx

```diff
@@ -14,93 +15,179 @@ function SeriesDetailsLinks(props: SeriesDetailsLinksProps) {
-          kind={kinds.INFO}
-          size={sizes.LARGE}
-        >
-          Trakt
-        </Label>
-      </Link>
-
-      {tvMazeId ? (
-        <Link
-          className={styles.link}
-          to={`https://www.tvmaze.com/shows/${tvMazeId}/_`}
-        >
-          <Label
-            className={styles.linkLabel}
-            kind={kinds.INFO}
-            size={sizes.LARGE}
-          >
-            TV Maze
-          </Label>
-        </Link>
-      ) : null}
-
-      {imdbId ? (
-        <>
-          <Link
-            className={styles.link}
-            to={`https://imdb.com/title/${imdbId}/`}
-          >
-            <Label
-              className={styles.linkLabel}
-              kind={kinds.INFO}
-              size={sizes.LARGE}
-            >
-              IMDB
+    <>
+      <style>
+        {`
+          .series-links-row {
+            display: flex;
+            flex-direction: row;
+            gap: 12px;
+            flex-wrap: wrap;
+          }
+          .series-link-group {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            min-width: 100px;
+          }
+          .series-link-id-row {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+          }
+          @media (max-width: 600px) {
+            .series-links-row {
+              flex-direction: column;
+              gap: 12px;
+              align-items: stretch;
+            }
+            .series-link-group {
+              flex-direction: row;
+              align-items: center;
+              min-width: 0;
+              justify-content: flex-start;
+            }
+            .series-link-id-row {
+              margin-top: 0;
+              margin-left: 8px;
+            }
+          }
+        `}
+      </style>
+      <div className="series-links-row">
+        {/* TVDB */}
+        <div className="series-link-group">
+          <Link to={`https://www.thetvdb.com/?tab=series&id=${tvdbId}`}>
+            <Label kind={kinds.INFO} size={sizes.LARGE}>
+              The TVDB
             </Label>
           </Link>
-
-          <Link
-            className={styles.link}
-            to={`http://mdblist.com/show/${imdbId}`}
-          >
-            <Label
-              className={styles.linkLabel}
-              kind={kinds.INFO}
-              size={sizes.LARGE}
+          <div className="series-link-id-row">
+            <span>{tvdbId}</span>
+            <button
+              style={{ marginLeft: '4px', cursor: 'pointer' }}
+              type="button"
+              aria-label="Copy tvdbId"
+              onClick={() => navigator.clipboard.writeText(String(tvdbId))}
+              title="Copy tvdbId"
             >
-              MDBList
+              <Icon name={icons.CLIPBOARD} size={16} />
+            </button>
+          </div>
+        </div>
+
+        {/* Trakt */}
+        <div className="series-link-group">
```

Comment:
Instead of a bunch of duplication these should be extracted to a separate component.